dotcom-rendering/src/web/components/TextBlockComponent.tsx

```diff
@@ -77,23 +83,27 @@ const allowsDropCaps = (format: ArticleFormat) => {
 	}
 };
 
-const shouldShowDropCap = ({
-	format,
-	isFirstParagraph,
-	forceDropCap,
-}: {
-	format: ArticleFormat;
-	isFirstParagraph: boolean;
-	forceDropCap?: boolean;
-}): boolean => {
-	if (allowsDropCaps(format)) {
+const shouldShowDropCaps = (
```

Comment:
Instead of having `allowDropCaps` and some other rules about the place, I thought it was neater to just put all the reasons we would show drop caps in one method.